.github/workflows/frontend.yml

```diff
@@ -44,6 +44,8 @@ jobs:
           m2-cache-key: "cljs"
       - name: Compile CLJS
         run: yarn build-pure:cljs
+      - name: Find CSS variables that are used but never defined
```

Comment:
I added as a step in the `fe-lint task` as it seemed its natural place, it only adds a few seconds to CI